code/ui/components/src/syntaxhighlighter/syntaxhighlighter.stories.tsx

```diff
@@ -1,8 +1,13 @@
+import { ThemeProvider, ensure, themes } from '@storybook/theming';
+
 import type { ComponentProps } from 'react';
 import React from 'react';
-import { ThemeProvider, themes, ensure } from '@storybook/theming';
+import scss from 'react-syntax-highlighter/dist/esm/languages/prism/scss';
 import { SyntaxHighlighter } from './lazy-syntaxhighlighter';
 
+// Register custom language
+SyntaxHighlighter.registerLanguage('scss', scss);
```

Comment:
I think this should be done, as the story renders

Reply:
Ah, gotcha. Let me try that. 👍

Reply:
Realize I'm reading too quickly and did this in a custom _loader_ instead of _render_ func in https://github.com/storybookjs/storybook/pull/23166/commits/a9d0d223ac0e5dddd4c22cddbeb4d5d3df4239b6. The loader works, but let me know if you'd prefer I go the render route. Happy to rewrite.